include/sealib/dfs.h

```diff
@@ -38,57 +39,77 @@ namespace Sealib {
  */
 class DFS {
  public:
-  /**
-   * Run a standard depth-first search over a given graph. <br>
-   * EFFICIENCY: O(n+m) time, O(n log n) bits
-   * @param g graph G=(V,E) to iterate over
-   * @param preprocess to be executed before processing a node u
-   * @param preexplore to be executed before exploring an edge (u,v)
-   * @param postexplore to be executed after exploring an edge (u,v)
-   * @param postprocess to be executed after processing a node u
-   * @author Simon Heuser
-   */
-  static void standardDFS(Graph *g, UserFunc1 preprocess, UserFunc2 preexplore,
-                          UserFunc2 postexplore, UserFunc1 postprocess);
-  /**
-   * Run a space-efficient depth-first search over a given graph. (Elmasry,
-   * Hagerup and Kammer; 2015) <br>
-   * EFFICIENCY: O((n+m) log n) time, O((log3 + ε) n) bits
-   * @param g graph G=(V,E) to iterate over
-   * @param preprocess to be executed before processing a node u
-   * @param preexplore to be executed before exploring an edge (u,v)
-   * @param postexplore to be executed after exploring an edge (u,v)
-   * @param postprocess to be executed after processing a node u
-   * @author Simon Heuser
-   */
-  static void nBitDFS(Graph *g, UserFunc1 preprocess, UserFunc2 preexplore,
-                      UserFunc2 postexplore, UserFunc1 postprocess);
+    /**
+     * Run a standard depth-first search over a given graph. <br>
+     * EFFICIENCY: O(n+m) time, O(n log n) bits
+     * @param g graph G=(V,E) to iterate over
+     * @param preprocess to be executed before processing a node u
+     * @param preexplore to be executed before exploring an edge (u,v)
+     * @param postexplore to be executed after exploring an edge (u,v)
+     * @param postprocess to be executed after processing a node u
+     * @author Simon Heuser
+     */
+    static void standardDFS(Graph *g, UserFunc1 preprocess,
+                            UserFunc2 preexplore, UserFunc2 postexplore,
+                            UserFunc1 postprocess);
+    /**
+     * Run a space-efficient depth-first search over a given graph. (Elmasry,
+     * Hagerup and Kammer; 2015) <br>
+     * EFFICIENCY: O((n+m) log n) time, O((log3 + ε) n) bits
+     * @param g graph G=(V,E) to iterate over
+     * @param preprocess to be executed before processing a node u
+     * @param preexplore to be executed before exploring an edge (u,v)
+     * @param postexplore to be executed after exploring an edge (u,v)
+     * @param postprocess to be executed after processing a node u
+     * @author Simon Heuser
+     */
+    static void nBitDFS(Graph *g, UserFunc1 preprocess, UserFunc2 preexplore,
+                        UserFunc2 postexplore, UserFunc1 postprocess);
```

Comment:
Can not see any difference in the code, why have you modified these lines?

Reply:
I added myself as author.

Reply:
Disable auto format for not modified lines.